lib/test-jig.js

```diff
@@ -16,7 +16,10 @@ function TestJig(rules, testData) {
 
 TestJig.prototype._lookupAuth = function(auth) {
 
-  if (typeof auth === 'string') {
+  if (this.users[auth] == null) {
```

Comment:
convention for this repo is to use `===` for all comparison; can you please change this?